components/sankey/elements/links.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Links
+page_title: Sankey Links
+description: Links in the Sankey Diagram for Blazor.
+slug: sankey-links
+tags: telerik,blazor,sankey,diagram,chart,links
+published: True
+position: 3
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+# Sankey Links
+
+The Sankey Diagram Links are the lines that connect the [nodes]({%slug sankey-nodes%}) to each other. The greater the value of the link, the greater the width of the link will be. This article explains how to customize the links in the UI for Blazor Sankey Diagram.
+
+>tip Each setting applies to all links in the Sankey Diagram. If you want to provide different settings for the separate links, [customize them through the data source]({%slug sankey-data-binding%}#customize-elements-through-data).
+
+## Basic Customization
+
+To customize the nodes, declare the `<SankeyLinks>` tag as a direct child of `<TelerikSankey>`. The `<SankeyLinks>` tag exposes the following parameters:
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --------- | ---- | ----------- |
+| `ColorType` | `SankeyLinksColorType` enum <br /> (`SankeyLinksColorType.Static`) | The origin of the link color: <ul><li>`Static` - the link color is based on the `Color` property;</li><li>`Source` - the link color is based on the source node color; </li><li>`Target` - the link color is based on the target node color; </li></ul> |
+| `Color` | `string` | The color of the links. Applies when `ColorType="@SankeyLinksColorType.Static"`. |
```

Comment:
There is some default gray value that we can mention (or probably black color + some opacity)